lib.ps1

```diff
@@ -1,7 +1,7 @@
-$GIT_FOLDER = "platform-contributor-all\git"
+$GIT_FOLDER = "D:\Dev\Eclipse\eclipse-contrib\git"

```

Comment:
Please don't put your own values here, they should be kept local (for your own usage only)

Reply:
Solved in 46dcd3b37f4b18beefd6db44ad98e97e68501248